lightning/config/config.go

```diff
@@ -140,17 +140,28 @@ type CSVConfig struct {
 }
 
 type MydumperRuntime struct {
-	ReadBlockSize    int64     `toml:"read-block-size" json:"read-block-size"`
-	BatchSize        int64     `toml:"batch-size" json:"batch-size"`
-	BatchImportRatio float64   `toml:"batch-import-ratio" json:"batch-import-ratio"`
-	SourceDir        string    `toml:"data-source-dir" json:"data-source-dir"`
-	NoSchema         bool      `toml:"no-schema" json:"no-schema"`
-	CharacterSet     string    `toml:"character-set" json:"character-set"`
-	CSV              CSVConfig `toml:"csv" json:"csv"`
-	CaseSensitive    bool      `toml:"case-sensitive" json:"case-sensitive"`
-	StrictFormat     bool      `toml:"strict-format" json:"strict-format"`
-	MaxRegionSize    int64     `toml:"max-region-size" json:"max-region-size"`
-	Filter           []string  `toml:"filter" json:"filter"`
+	ReadBlockSize    int64            `toml:"read-block-size" json:"read-block-size"`
+	BatchSize        int64            `toml:"batch-size" json:"batch-size"`
+	BatchImportRatio float64          `toml:"batch-import-ratio" json:"batch-import-ratio"`
+	SourceDir        string           `toml:"data-source-dir" json:"data-source-dir"`
+	NoSchema         bool             `toml:"no-schema" json:"no-schema"`
+	CharacterSet     string           `toml:"character-set" json:"character-set"`
+	CSV              CSVConfig        `toml:"csv" json:"csv"`
+	CaseSensitive    bool             `toml:"case-sensitive" json:"case-sensitive"`
+	StrictFormat     bool             `toml:"strict-format" json:"strict-format"`
+	MaxRegionSize    int64            `toml:"max-region-size" json:"max-region-size"`
+	Filter           []string         `toml:"filter" json:"filter"`
+	FileRouters      []*FileRouteRule `toml:"files" json:"files"`
+	DefaultFileRules bool             `toml:"default-file-rules" json:"default-file-rules"`
```

Comment:
yes, I considered this but forgot to update the code. It can be done in adjustConfig. But I think in some case, user need to add some rule in addition to the default rule, thus set this to true will be convenient.

I will update the logic to set the default value to false and enable it `len(FileRouters) == 0`.  And user can explicitly enable it.